src/main/java/spoon/reflect/declaration/CtField.java

```diff
@@ -35,4 +35,9 @@
 	 * @see spoon.reflect.declaration.CtNamedElement#getReference()
 	 */
 	CtFieldReference<T> getReference();
+
+	/**
+	 * Replaces this element by another one.
+	 */
+	<R extends T> void replace(CtField<R> element);
```

Comment:
It may be useful in some cases. We may add a precondition on the name and/or the type.